lib/components/InputLoader/inputLoader.scss

```diff
@@ -0,0 +1,40 @@
+.input-loader {
+  display: flex;
+  align-items: center;
+  position: absolute;
+  width: 36px;
+  height: 20px;
+}
```

Comment:
Better to user display: inline-block: If the .input-loader needs to adjust its size based on content. display: flex might be redundant here.

Reply:
We need `display: flex` to be able to align the loader with `align-items: center`,
and also we don't really need `.input-loader` to adjust its size based on the content since its content is always the same